dbt-cta/stackadapt/models/1_cta_incremental/_1_cta_incremental__models.yml

```diff
@@ -0,0 +1,465 @@
+        description: ''
+      - name: cvr
+        description: ''
+      - name: imp
+        description: ''
+      - name: ltr
+        description: ''
+      - name: atos
+        description: ''
+      - name: conv
+        description: ''
+      - name: cost
+        description: ''
+      - name: date
+        description: ''
+      - name: ecpa
+        description: ''
+      - name: ecpc
+        description: ''
+      - name: ecpe
+        description: ''
+      - name: ecpm
+        description: ''
+      - name: ecpv
+        description: ''
+      - name: rcpc
+        description: ''
+      - name: rcpe
+        description: ''
+      - name: rcpm
+        description: ''
+      - name: roas
+        description: ''
+      - name: click
+        description: ''
+      - name: ecpcl
+        description: ''
+      - name: rcpcl
+        description: ''
+      - name: profit
+        description: ''
+      - name: s_conv
+        description: ''
+      - name: conv_ip
+        description: ''
+      - name: revenue
+        description: ''
+      - name: vcomp_0
+        description: ''
+      - name: conv_rev
+        description: ''
+      - name: vcomp_25
+        description: ''
+      - name: vcomp_50
+        description: ''
+      - name: vcomp_75
+        description: ''
+      - name: vcomp_95
+        description: ''
+      - name: line_item
+        description: ''
+      - name: page_time
+        description: ''
+      - name: uniq_conv
+        description: ''
+      - name: uniq_ecpa
+        description: ''
+      - name: atos_units
+        description: ''
+      - name: conv_click
+        description: ''
+      - name: page_start
+        description: ''
+      - name: reach
+        description: ''
+      - name: vcomp_rate
+        description: ''
+      - name: engage_rate
+        description: ''
+      - name: tp_cpc_cost
+        description: ''
+      - name: tp_cpm_cost
+        description: ''
+      - name: unique_conv
+        description: ''
+      - name: view_percent
+        description: ''
+      - name: page_time_15s
+        description: ''
+      - name: sub_advertiser
+        description: ''
+      - name: page_time_units
+        description: ''
+      - name: conv_imp_derived
+        description: ''
+      - name: conv_imp_time_avg
+        description: ''
+      - name: conv_click_time_avg
+        description: ''
+      - name: frequency
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+        tests:
+          - not_null
+          - unique
+      - name: _airbyte_emitted_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_normalized_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_account_line_items_stats_hashid
```

Comment:
maybe a unique check on the hash id would be good? Since the `_airbyte_ab_id` will always be unique

Reply:
oh! Good catch, just added that in

Reply:
:catjam: